### ISSUE TITLE
Can't create install dir on osX

### ISSUE THREAD

**Issue by paynito** [roddhjav/pass-update#5]
→ sudo make install
Password:
install: mkdir /usr/lib/password-store: Operation not permitted
make: *** [install] Error 71

**Comment by roddhjav** [comment]
Thanks for this feedback. On OS X you need to specify a prefix:
```
git clone https://github.com/roddhjav/pass-update/
cd pass-update
sudo make install PREFIX=/usr/local
```

I'll add doc in the README soon.

**Comment by roddhjav** [comment]
Fixed by cc34b422ff46428737b919b2654c3b58f4eac1a7
